tcms/telemetry/templates/telemetry/testing/breakdown.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block title %}{% trans "Testing Breakdown" %}{% endblock %}
+
+{% block contents %}
+
+    <div class="container-fluid container-cards-pf">
+
+        <form class="form-horizontal">
+            <div class="form-group">
+                <label for="id_select_product" class="col-md-1 col-lg-1">
+                    {% trans "Product" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="product" id="id_select_product" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for product in products %}
+                            <option value="{{ product.pk }}">
+                                {{ product.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+
+                <label for="id_select_test_plan" class="col-md-1 col-lg-1">
+                    {% trans "Test Plan" %}:
+                </label>
+                <div class="col-md-3">
+                    <select name="test_plan" id="id_select_test_plan" class="form-control selectpicker">
+                        <option value="">----------</option>
+                        {% for test_plan in test_plans %}
+                            <option value="{{ test_plan.pk }}">
+                                {{ test_plan.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </div>
+        </form>
+
+        <div class="col-md-3" style="padding: 15px; margin: 15px; text-align: center">
+            <div style="text-align: center; font-weight: bold"> {% trans "Automated/ Manual" %}</div>
+            <div id="total"></div>
+            <div> {% trans "Total" %}</div>
+
+            <div id="automated-bar" style="padding: 10px; font-weight: bold"></div>
+        </div>
+
+        <div class="col-md-4" style="padding: 15px; margin: 15px; text-align: center">
```

Comment:
Remove styling, especially padding and margin. This should be dealth with using HTML and CSS styles on the widgets.